app/controllers/kaui/audit_logs_controller.rb

```diff
@@ -8,7 +8,8 @@ def index
 
 
     formatter = lambda do |log|
-      object_id_text = view_context.content_tag(:span, view_context.truncate_uuid(log.object_id), title: log.object_id )
+      object_id_text = view_context.object_id_popover(log.object_id)
```

Comment:
When the truncated UUID is a link, the popover shouldn't display the link (otherwise, you cannot click on the clipboard).

![image](https://user-images.githubusercontent.com/51940/40543854-10f31968-5fdb-11e8-9e4a-ca0d61c9e4d9.png)